app/core/usePrintPDF/usePrintPDF.js

```
@@ -0,0 +1,187 @@
+    let printTriggerTimeout;
+
+    // Whenever a step is successfully completed, this effect triggers the next step in the sequence.
+
+    switch(lastCompletedStep) {
+      case STATUS.CLEARING_CACHE:
+        dispatch(actions.worker.removeGeneratedPDFS());
+        dispatch(actions.worker.dataWorkerRemoveDataRequest(null, patientId));
+        break;
+
+      case STATUS.FETCHING_MODAL_DATA:
+        const initialFetchOpts = getInitialFetchOpts();
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)